packages/core/src/app/payment/Payment.tsx

```diff
@@ -207,6 +207,7 @@ class Payment extends Component<
                             }
                             shouldHidePaymentSubmitButton={
                                 (uniqueSelectedMethodId &&
+                                    !rest.isStoreCreditApplied &&
```

Comment:
Have we tested if this condition causes issues with other providers?

Reply:
I will mention it in the comments for task.